dropwizard-bom/pom.xml

```diff
@@ -140,7 +140,13 @@
             <dependency>
                 <groupId>org.liquibase</groupId>
                 <artifactId>liquibase-core</artifactId>
-                <version>3.4.2</version>
+                <version>3.5.0</version>
+                <exclusions>
```

Comment:
Necessary because of:
```
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce) @ dropwizard-migrations ---
[WARNING]
Dependency convergence error for org.yaml:snakeyaml:1.15 paths to dependency are:
+-io.dropwizard:dropwizard-migrations:1.0.0-rc3-SNAPSHOT
  +-io.dropwizard:dropwizard-db:1.0.0-rc3-SNAPSHOT
    +-io.dropwizard:dropwizard-core:1.0.0-rc3-SNAPSHOT
      +-io.dropwizard:dropwizard-configuration:1.0.0-rc3-SNAPSHOT
        +-com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.7.3
          +-org.yaml:snakeyaml:1.15
and
+-io.dropwizard:dropwizard-migrations:1.0.0-rc3-SNAPSHOT
  +-org.liquibase:liquibase-core:3.5.0
    +-org.yaml:snakeyaml:1.13

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability the error(s) are [
Dependency convergence error for org.yaml:snakeyaml:1.15 paths to dependency are:
+-io.dropwizard:dropwizard-migrations:1.0.0-rc3-SNAPSHOT
  +-io.dropwizard:dropwizard-db:1.0.0-rc3-SNAPSHOT
    +-io.dropwizard:dropwizard-core:1.0.0-rc3-SNAPSHOT
      +-io.dropwizard:dropwizard-configuration:1.0.0-rc3-SNAPSHOT
        +-com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.7.3
          +-org.yaml:snakeyaml:1.15
and
+-io.dropwizard:dropwizard-migrations:1.0.0-rc3-SNAPSHOT
  +-org.liquibase:liquibase-core:3.5.0
    +-org.yaml:snakeyaml:1.13
```